.github/workflows/mbed_ci_stress_test.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2020-2021 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Mbed CI stress test
+
+on:
+  schedule:
+    # Triggers the workflow every 30 minutes:
+    - cron:  '*/30 * * * *'
+
+jobs:
+    trigger-ci-actions:
+        runs-on: ubuntu-latest
+        steps:
+            - run: echo "Trigger Mbed CI actions"
```

Comment:
No new line at the end of the file - it's good to add it now instead of being block by upstream's restyler.